turbinia/workers/plaso.py

```diff
@@ -130,10 +121,8 @@ def run(self, evidence, result):
 
     cmd.extend(['--temporary_directory', self.tmp_dir])
     cmd.extend(['--logfile', plaso_log])
-    cmd.extend([plaso_file, evidence.local_path])
-
+    cmd.extend([plaso_file, evidence.source_path])
```

Comment:
I think you still want `evidence.source_path` here since that's the last path that is set in the pre-processors.  It's possible that when you started this there was a bug that needed it to be set to `local_path`, but that should be fixed now.

Reply:
This is currently set to source_path, did you mean the reverse?